src/SDKs/CognitiveServices/dataPlane/Vision/Vision.Tests/Microsoft.Azure.CognitiveServices.Vision.Tests.csproj

```diff
@@ -43,6 +43,9 @@
     <None Update="TestImages\Gates4.jpg">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Update="TestImages\house.jpg">
```

Comment:
@DavidLiCIG please replace explicit includes of .jpg with wild card includes similar to what you did for *.json